docket/application.py

```diff
@@ -1,21 +1,25 @@
 #!/usr/bin/env python
 from flask import Flask, jsonify
-from flask_script import Manager
 from celery import Celery
 
 from common.exceptions import HTTPException
 
-import yaml
-import os
+from config import Config
 import logging
 
+# !! can't use celery decorator here - we haven't made it yet
```

Comment:
So, what's the result? Are we doing this or not? If not, let's remove this code.

Reply:
yep that needs to go.